app/src/main/res/layout/product_image.xml

```diff
@@ -1,5 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent" android:layout_height="match_parent">
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <TextView
+        android:id="@+id/tvDescription"
```

Comment:
You were using ProductsAdapater till yesterday. It requires a text view with tvDescription. So I added (copied) from another layout.